arches_modular_reports/models.py

```diff
@@ -34,6 +35,12 @@ class ReportConfig(models.Model):
     class Meta:
         managed = True
         db_table = "arches_modular_report_config"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["graph", "slug"],
+                name="unique_slug_graph",
```

Comment:
We can't enforce uniqueness here because a single graph can have any number of configs. In the case of rascolls - a config for the default report and a second for the search result details